packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -223,7 +225,7 @@ class _CupertinoSliderState extends State<CupertinoSlider> with TickerProviderSt
     return new _CupertinoSliderRenderObjectWidget(
       value: (widget.value - widget.min) / (widget.max - widget.min),
       divisions: widget.divisions,
-      activeColor: widget.activeColor,
+      activeColor: widget.activeColor ?? CupertinoColors.activeBlue,
```

Comment:
Why move this from parameters?  Seems to be the same effect, just more difficult to find...

Reply:
fwiw generally we much prefer defaults to be explicitly in the argument values, since that's much more self-documenting.